test/screens/v2/widget_instance/current_elevator_closed_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule Screens.V2.WidgetInstance.CurrentElevatorClosedTest do
+  use ExUnit.Case, async: true
+
+  alias Screens.V2.WidgetInstance
+  alias Screens.V2.WidgetInstance.CurrentElevatorClosed
+  alias ScreensConfig.V2.Elevator
+
+  setup do
+    %{
+      instance: %CurrentElevatorClosed{
+        app_params:
+          struct(Elevator,
+            elevator_id: "111",
+            alternate_direction_text: "Test",
+            accessible_path_direction_arrow: :n
+          ),
+        closure: %CurrentElevatorClosed.Closure{
+          description: "Test Alert Description",
+          elevator_name: "Test Elevator",
+          elevator_id: "111",
+          id: "1",
+          header_text: "Test Alert Header"
+        }
+      }
+    }
+  end
+
+  describe "priority/1" do
+    test "returns 1", %{instance: instance} do
+      assert [1] == WidgetInstance.priority(instance)
+    end
+  end
+
+  describe "serialize/1" do
+    test "returns map with id, closure, and alternate direction info", %{instance: instance} do
```

Comment:
I don't know if we really _need_ all these "it does the thing" tests for functions whose implementation is extremely basic... ideally the code paths would be covered by some higher-level test, but we have no such category of tests at the moment. Guess I won't make any specific recommendation here, just noting that it's an unfortunate situation.

Reply:
I have this thought every time I make tests for a new widget. Definitely a good idea to put end-to-end framework tests on some kind of radar. Not sure if that's big enough to be roadmap worthy or if a simple tech-debt task would be sufficient?